modules/javafx.graphics/src/main/java/javafx/css/CssMetaData.java

```diff
@@ -327,5 +328,23 @@ public int hashCode() {
             .append("}").toString();
     }
 
-
+    /**
+     * Utility method which combines {@code CssMetaData} items in one unmodifiable list with size equal
+     * to the number of items it holds.
+     *
+     * @param list the css metadata items, usually from the parent, must not be null
+     * @param items the additional items
+     * @return the unmodifiable list containing all of the items
+     *
+     * @since 22
+     */
+    public static List<CssMetaData<? extends Styleable, ?>> initStyleables(
```

Comment:
Have you considered a method name such as `concat` or `combine`? This method doesn't initialize styleable properties (nor their metadata).